doc/api/typescript.md

```diff
@@ -76,15 +76,15 @@ generation, and by replacing inline types with whitespace, Node.js can run
 TypeScript code without the need for source maps.
 
 Type stripping is compatible with most versions of TypeScript
-but we recommend version 5.7 or newer with the following `tsconfig.json` settings:
+but we recommend version 5.8 or newer with the following `tsconfig.json` settings:
 
 ```json
 {
   "compilerOptions": {
      "target": "esnext",
      "module": "nodenext",
```

Comment:
5.8 now supports `require(ESM)`, so this file technically won't work quite right for Node 20; does that matter? I assume not?

Reply:
My guess is there is no problem.

- Node 18/20 don't have TS support today.
- Node 18 ends LTS next month.
- Node 20 will get `require(ESM)` in the next release - it just landed.

Reply:
typescript wont be backported to v18 and v20 so its fine